src/components/ButtonPanel.js

```diff
@@ -1,7 +1,50 @@
-export default function ButtonPanel() {
-  return (
-    <h1>
-      Button Panel
-    </h1>
-  );
-}
+import React from 'react';
+import Button from './buttons/button';
+
+/* eslint-disable */
```

Comment:
- [ ] <b>REQUIRED:</b> You are not allowed to use `eslint-disable` unless it is totally obvious that you have to. it's like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). kindy try to fix the errors instead of ignoring them.